spec/models/thermometer_widget_spec.rb

```diff
@@ -1,41 +1,25 @@
 describe ThermometerWidget do
 
-  let(:content) { {
+  let(:params) { {
     goal: 10000,
     count: 500,
-    autoincrement: true
+    autoincrement: true,
+    page_display_order: 1
   } }
-  let(:string_content) { {
-    goal: "12345",
-    count: "45",
-    autoincrement: "0"
-  } }
```

Comment:
ahh. the problem I see with that is we can't query directly into the jsonb. As I understand it, postgres lets us do queries like "where content.count > 4000" but we lose that power if we store everything as strings